packages/next-auth/src/core/types.ts

```diff
@@ -15,6 +15,7 @@ import type { CookieSerializeOptions } from "cookie"
 import type { NextApiRequest, NextApiResponse } from "next"
 
 export type Awaitable<T> = T | PromiseLike<T>
+export type Falsy = null|undefined|false
```

Comment:
Let's not introduce a new type for this, only check for `null`. The other values don't make sense anyway.